src/reducers/cell-reducer.js

```diff
@@ -60,8 +182,17 @@ const cellReducer = (state = newNotebook(), action) => {
         { cells: moveCell(state.cells, getSelectedCellId(state), 'down') },
       )
 
-    case 'UPDATE_CELL_PROPERTIES':
+    case 'UPDATE_CELL_PROPERTIES': {
+      const approvedMultipleChanges = new Set(['skipInRunAll'])
```

Comment:
the const `approvedMultipleChanges` can go at the top of the file (it's kind of a global) and it should have a brief comment describing what it is. also i think it can probably just be a plain list rather than a set.